.github/actions/bootstrap/action.yml

```diff
@@ -481,3 +481,14 @@ runs:
         wget -O /tmp/oasdiff.tar.gz https://github.com/oasdiff/oasdiff/releases/download/v${{ env.OASDIFF_VER }}/oasdiff_${{ env.OASDIFF_VER }}_linux_amd64.tar.gz
         sudo tar xf /tmp/oasdiff.tar.gz -C /usr/local/bin oasdiff
         rm /tmp/oasdiff.tar.gz
+
+    - name: Install gh cli
```

Comment:
The GitHub CLI is automatically installed on all CI runners, per the [GitHub documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/using-github-cli-in-workflows):

> GitHub CLI is preinstalled on all GitHub-hosted runners. For each step that uses GitHub CLI, you must set an environment variable called GH_TOKEN to a token with the required scopes.

I have already used the `gh` CLI in a few workflows.  For [example](https://github.com/open-edge-platform/orch-ci/blob/main/.github/workflows/publish-documentation.yml#L317)